app/models/user.rb

```diff
@@ -30,4 +30,8 @@ def has_upvoted?(review)
   def has_downvoted?(review)
     has_voted?(review) && votes.find_by(review: review).value == -1
   end
+
+  def admin?
```

Comment:
This is built into Ruby for booleans. If admin is a boolean, you can call .admin? without defining it here.